### ISSUE TITLE
Add comment for new color palette_red_light

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#184]
See #178 

All colors in `_colors.scss` should have an inline comment to give _some_ indication about the other color they can be paired with to have a minimum color contrast ratio of 4.5:1
